### ISSUE TITLE
[Feature request] 5-star rating scale support

### ISSUE THREAD

**Issue by quietmansoath** [austinried/subtracks#37]
Just commenting as an itunes refugee with an eye on this project and as a navidrome fan. Navidorme has support for a 5-star rating scale system, and has just implemented smart playlists. Please consider adding these to your app down the line!

**Comment by RedKage** [comment]
Same here, I have no android app which support 5 stars ratings for subsonic ...
There was one app though, can't remember which, and it had a 5 vs 1 slider or something in the setting somewhere.
But that app was broken.

**Comment by cquike** [comment]
Hi,
@RedKage : ultrasonic in Android supports 5 star ratings.

**Comment by jsrol** [comment]
As 2.0 is getting alpha feedback, I'd like to bump this feature request which is a must for a lot of users, especially with very large libraries. Current star rating should be replaced by the "heart" button instead.

**Comment by austinried** [comment]
> Current star rating should be replaced by the "heart" button instead.

Good point, I have had this as a star since the beginning since that's just what subsonic calls it. I'm not sure if this will make it in the alpha but in order to make sure it's not confusing in the future, it's probably better for "favorite" to not use the same icon as the rating scale.
